guides/V1_ActionsEvents.md

```diff
@@ -0,0 +1,212 @@
+
+## Listen to Events
+
+In our programming model we refer to Events in general, as those triggered based on changes to the data model regardless of the source of that change (ie. a human using a client, an app on a schedule, etc...). One exception is the actionSelected event which makes no changes to the data model and is referred to later in this guide.
+
+The common theme is that typically, an app that handles these events need not present a user experience to a user and can operate in a "headless" or "no-UI needed" fashion.
+
+Handling these events was also known as apps "listening to a conversation", but as has been established there are many other types of events.
+
+Please read [Listen to Events](https://github.com/watsonwork/watsonwork-developer-docs/blob/master/guides/V1_wwsg_Webhooks.md) here for information on handling these types of events.
+
+## Handling Client Actions
+
+Watson Work Services allows your App to provide for actions issued from a Watson Work Services client (like Watson Workspace). 
+Here your App will be invoked directly from the client app and can provide a user experience to guide a user through the 
+fulfillment of that action. This is accomplished in one of two ways, using Commands or Client Action Handlers. 
+
+Watson Work Services allows your App to provide handlers for user actions issued from a Watson Work Services client (like Watson Workspace). User actions can be classified as:
+
+1. User actions which can be handled by an app with an indirect user experience where the app developer declares simpler UI constructs for the Watson Work services client to then generate UI and present to the user. Example: /commands
+2. User actions which can be handled by an app providing a direct user experience where the app developer codes client specific UI. Here your App will be invoked directly from the Watson Work Services client (such as Workspace) and it can provide a user experience to guide a user through the fulfillment of that action. Example: App Configuration
+
+### Implicit Action Handler
+
+These type of user actions are expected to be handled without the need for apps to provide UI code. The app is notified via a webhook and typically provides a declaration of a user experience for the Work Services client to interpret and present to the user. Examples are:
```

Comment:
Clarification on which webhook since we said about `actionSelected` would be visited later in the guide.

from:

>These type of user actions are expected to be handled without the need for apps to provide UI code. The app is notified via a webhook and typically provides a declaration of a user experience for the Work Services client to interpret and present to the user

to:

These type of user actions are expected to be handled without the need for apps to provide UI code. The app is notified via the `actionSelected` webhook event (currently part of the `annotation-added` event, but soon to be promoted into its own top level event)  and typically provides a declaration of a user experience for the Work Services client to interpret and present to the user (via the createTargetedMessage mutation)